CONTRIBUTING.md

```diff
@@ -1,65 +1,231 @@
+   Remember that all comments are public. Keep your replies civil and relevant.
+
+1. When your PR is ready for another round of feedback,
+   re-request a review or write a comment with `PTAL @[reviewer]`.
+   (PTAL means _please take another look_.)
+
+1. Once your PR is approved, a reviewer [merges it][merges-pr].
+
+## Reviewers
+
+### General guidelines for reviews
+
+* Technical review comes before content review, or "copy", review.
+* Lean toward velocity. Aim for _improvement_ over absolute _perfection_.
+* Check the whole page. Sometimes new content contradicts unchanged
+  content elsewhere on the same page.
+* Prefer giving an overall note when there's a repeated problem,
+  rather than commenting on every instance.
+* Try to **approve** with suggestions wherever possible.
+  Use **request changes** only when the change can harm an end user's work or
+  violates [Flutter's Code of Conduct][code-of-conduct].
+
+### Review the technical concerns
+
+In general, follow these practices.
+
+* Critique the _technical aspects_ of the PR.
+  If the writer makes a change that's technically correct but could be done
+  differently, _approve the PR_.
+* Provide suggested changes with corrected copy to make your change clear.
+
+To conduct a consistent technical review in the Flutter website:
+
+1. [Review a PR][review-pr] when it meets the following conditions:
+
+   1. The writer [requested your review][request-review] of the PR.
+
+   1. The writer applied the [**review.tech**][review-tech-label]
+      label to the PR.
+
+   1. The writer didn't set the PR to [Draft mode][set-draft].
+      You can provide feedback to a draft issue,
+      but you should understand that the author's still working on the draft.
+
+1. Validate technical accuracy.
+
+   1. Check any step-by-step procedures and code examples.
+
+   1. If code examples have been added or changed,
+      verify that new code is included in CI and that tests are passing.
+
+   1. Verify that the code remains consistent with the
+      [Flutter Tree Hygiene guide][flutter-tree]
+      and the [Flutter code style guide][flutter-style].
+
+1. When [adding comments][add-comments],
+   state explicitly if you're making a suggestion or
+   pointing out something that needs to change.
+
+1. When completing a review,
+   favor [approving the PR][approve-pr] rather than requesting changes,
+   whenever possible.
+
+   1. If all of your comments were suggestions rather than issues,
+      approve the PR and trust the submitter to read and consider them
+      before merging.
+      Click **Approve** and change the label from
+      [**review.tech**][review-tech-label] to
+      [**review.copy**][review-copy-label].
+
+   1. If the PR needs concrete changes before you can approve,
+      note the reason why,
+      click **Comment** and change the label from
+      [**review.tech**][review-tech-label] to
+      [**review.await-update**][review-await-label].
+
+### Review the content quality
```

Comment:
I think these headings should be more clear. It's not clear from the headings or the contents in each section that these are two separate types of reviews, and if you're requested as a reviewer, you're likely only responsible for one (correct?) Something like

```suggestion
### Copy review

If you're requested for a _copy_ review... (follow these practices)
```

Reply:
That's tricky. I'm trying to keep headings parallel.